src/components/PageSidebar.vue

```diff
@@ -2,27 +2,29 @@
 	<NcAppSidebar ref="sidebar"
 		:title="title"
 		@close="close">
-		<NcAppSidebarTab id="backlinks"
+		<NcAppSidebarTab id="attachments"
```

Comment:
Is the attachment sidebar supported on public shares? From the component below I assumed not, then we may want to hide it.